Tests/SwiftDocCTests/Model/SemaToRenderNodeTests.swift

```diff
@@ -2840,6 +2840,63 @@ Document @1:1-11:19
         XCTAssertTrue(reference.isDeprecated)
     }
     
+    func testSymbolLinkWorkInMultipleLanguages() throws {
```

Comment:
`SemaToRenderNodeTests` is becoming quite large, so we created `SemaToRenderNodeMixedLanguageTests.swift` for multi-language tests. Could we please move this test there?

Reply:
Yes. I'll move this test there.